draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx

```diff
@@ -126,6 +126,10 @@ const Tooltip = ({
     </AppearanceAnim>
   )
 
+  const portalSelectorElement: Element | null = portalSelector
+    ? document.querySelector(portalSelector)
+    : null
```

Comment:
Thanks for fixing this. Not a big deal, but I would probably check if this `querySelector` fails, to show a console warning as a result. Otherwise if we fail silently, a developer could be confused as to why their tooltip isn't showing in the correct location in the DOM tree.